src/util.py

```diff
@@ -260,12 +265,127 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def plotCLabels(da, contours, transform, ax, proj, Clevels=[], lowClevels=[], highClevels=[], cfs=14, lfs=22, hfs=22):
+
+    """
+    Utility function to plot contour labels. Regular contour labels will be plotted using the built-in matplotlib
+    clabel function. High/Low contour labels will be plotted using text boxes for more accurate label values 
+    and placement.
+
+    Args:
+
+        da: (:class:`xarray.DataArray`):
+            Xarray data array containing the lat, lon, and field variable data values.
+
+        contours (:class:`cartopy.mpl.contour.GeoContourSet`):
+            Contour set that is being labeled.
+
+        transform (:class:`cartopy._crs`):
+            Projection that the input coordinates (in GPS form of lon, lat) should be transformed to
```

Comment:
Can you clarify the difference between `transform` and `proj`? Shouldn't the data be transformed into `proj` since that is the projection of the axis? What's the purpose of `transform`? My confusion stems from how the comment here states "Projection that the input coordinates should be transformed *to*" versus "transformed *from*"

Reply:
In some cases, they are not the same. For example, when using this function on sat1/sat2 the "proj" is orthographic (with extent set) but the transform would be ccrs.Geodetic() to make the labels show up at the appropriate locations. I'll change the comments to make that clearer.